### DISCUSSION TITLE
'Local' ? 'etc' ?

### DISCUSSION THREAD

**Discussion by ccgguy** [NikitaCern/TaleSpire_Generator#4] [Q&A] [answered]
1.Clone the repo to your local
**where is local? I assume its just to the machine. 
If not, what is the root folder to clone repo to? steamapps\common\TaleSpire?

2.Locate index.json file in Talespire installation folder. Most likely it is in: ..\Steam\steamapps\common\TaleSpire\Taleweaver\d71427a1-5535-4fa7-82d7-4ca1e75edbfd 
**correct.

3. Place the index.json in the etc folder of the cloned repository
**'etc' is not in the cloned repo? I can make 'etc' and copy 'index.json' to the created 'etc'... 
** I think python uses CWD (Current working directory) so I just created 'etc' and dropped index.json into it, this didn't work.
** I then tried modifying the setup code to use a full path to etc/index.json E.G. E:/steamapps/common/.......etc/index.json, this didn't work.

4. Run the setup script with the command:
** haven't made it this far but it makes sense

**Comment by SkogrKrellr** [comment, marked as answer]
1. Local, as in your local machine, it can be cloned anywhere you want.
3. Fix should be in place now
